src/System.Net.Mail/src/System.Net.Mail.csproj

```diff
@@ -144,5 +144,14 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
+    <ProjectReference Include="..\..\System.Net.Requests\ref\System.Net.Requests.csproj">
+      <UndefineProperties>TargetGroup;OSGroup</UndefineProperties>
```

Comment:
Otherwise it results in bin clashes.  The refs need to be built without OSGroup and TargetGroup set.